### ISSUE TITLE
Arch install example

### ISSUE THREAD

**Issue by C0rn3j** [eliostvs/tomate-gtk#23]
Currently you use yaourt to provide an example for downloading on Arch based distros.

This is not a safe way to install software, so I'd recommend to just switch the example to "pacaur -S tomate-gtk", as that is a much better option.

https://www.reddit.com/r/archlinux/comments/5s7ojo/yaourt_explain_to_me_why_its_bad/

**Comment by C0rn3j** [comment]
Between the time I posted that and you changed it Pacaur went from maintained, to unmaintained to dead.

Would probably be for the best to just leave a link to the AUR page.

**Comment by eliostvs** [comment]
Yes, you are right. I just released a patch changing the text. Thanks again.
